src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/aix/CallArranger.java

```diff
@@ -50,14 +50,11 @@ public class CallArranger {
 
 	/* Replace DowncallLinker in OpenJDK with the implementation of DowncallLinker specific to OpenJ9 */
 	public static MethodHandle arrangeDowncall(MethodType mt, FunctionDescriptor cDesc, LinkerOptions options) {
-		// MethodHandle handle = DowncallLinker.getBoundMethodHandle(mt, cDesc, options);
-		// return handle;
-		return null;
+		return DowncallLinker.getBoundMethodHandle(mt, cDesc, options);
 	}
 
 	/* Replace UpcallLinker in OpenJDK with the implementation of UpcallLinker specific to OpenJ9 */
-	public static MemorySegment arrangeUpcall(MethodHandle target, MethodType mt, FunctionDescriptor cDesc, SegmentScope session) {
-		// return UpcallLinker.make(target, mt, cDesc, session);
-		return null;
+	public UpcallStubFactory arrangeUpcall(MethodType mt, FunctionDescriptor cDesc) {
```

Comment:
This won't compile: the method must be `static` and an import for `UpcallStubFactory` is needed:
```
src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/aix/CallArranger.java:57: error: cannot find symbol
        public UpcallStubFactory arrangeUpcall(MethodType mt, FunctionDescriptor cDesc) {
               ^
  symbol:   class UpcallStubFactory
  location: class CallArranger
```
Same problems in `ppc64/sysv/CallArranger.java` and `s390x/sysv/CallArranger.java`.

Reply:
Fixed, tested in [an internal personal build](https://hyc-runtimes-jenkins.swg-devops.com/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/16089)
AIX had an error
```
17:12:18  src/java.desktop/share/native/libharfbuzz/hb-algs.hh:882:10: error: 1540-2993 The builtin "__builtin_mul_overflow" is not supported.
17:12:18    return __builtin_mul_overflow (count, size, result);
17:12:18           ^
17:12:18  1 error generated.
```
It seems a build machine issue.